extensions/amp-position-observer/0.1/amp-position-observer.js

```diff
@@ -360,8 +360,8 @@ export class AmpVisibilityObserver extends AMP.BaseElement {
       scene = this.element.parentNode;
     }
     // Hoist body to documentElement.
-    if (this.getAmpDoc().getBody() == scene) {
-      scene = this.win.document.documentElement;
+    if (scene && scene.tagName === 'BODY') {
```

Comment:
Should add a comment that we need to check this way to handle friendly iframes, since the intent here is not clear.

Reply:
Added comments.